explainer.md

```diff
@@ -148,19 +151,3 @@ This will correctly format the price in the given locale (which should be set to
 *   Do we need to support [pending transactions](https://developer.android.com/google/play/billing/billing_library_overview#pending)? (i.e., when your app starts, you’re expected to query pending transactions which were made out-of-app, and acknowledge them).
     *   In the Play Billing backend, this means you’re supposed to call [BillingClient.queryPurchases](https://developer.android.com/reference/com/android/billingclient/api/BillingClient#querypurchases) to get the list of pending unacknowledged transactions.
     *   See [this post](https://android-developers.googleblog.com/2019/06/advanced-in-app-billing-handling.html) for details.
```

Comment:
I think you should not delete this discussion context, but maybe move it into a "Resolved issues" section at the end. In fact, I think (later) we should introduce more context for some of the decisions we're making here, with clear rationale, to avoid having to re-litigate these arguments again.

Reply:
Done.